var/spack/repos/builtin/packages/py-cmyt/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class PyCmyt(PythonPackage):
+    """Matplotlib colormaps from the yt project !"""
+    homepage = "https://yt-project.org"
+    pypi = "cmyt/cmyt-1.0.4.tar.gz"
+    git = "https://github.com/yt-project/cmyt.git"
+
+    maintainers = ['qobilidop', 'charmoniumq']
+
+    version("main", branch="main")
+
+    version("1.0.4", "c833ed24eadca8c788e59ce27119d26b2e7d4c6af83293e69385412629294b2a")
```

Comment:
PyPI checksums are different than GitHub, you'll need to recompute these. You can use `spack checksum py-cmyt` to do this.